pkg/module/oci/client.go

```diff
@@ -0,0 +1,213 @@
+package oci
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/containerd/containerd/content"
+	"github.com/containerd/containerd/errdefs"
+	"github.com/containerd/containerd/images"
+	"github.com/containerd/containerd/remotes"
+	"github.com/containerd/containerd/remotes/docker"
+	"github.com/opencontainers/go-digest"
+	ocispecv1 "github.com/opencontainers/image-spec/specs-go/v1"
+)
+
+const (
+	DefaultTimeout = 10 * time.Second
+)
+
+type Client interface {
+	// GetManifest returns the ocispec Manifest for a reference
```

Comment:
```suggestion
	// GetManifest returns the ocispec manifest for a reference
```
reference is in lower case in line 30 and 33, so why capitalized here?